packages/design-system-web/src/components/geov-entity-properties/geov-entity-properties.tsx

```diff
@@ -0,0 +1,206 @@
+import { Component, Host, h, Prop, State } from '@stencil/core';
+import { sparqlJson } from '../../lib/sparqlJson';
+
+const qrOntomeURI = (id: string) => `
```

Comment:
the whole query is not needed, since the rdf:type can be queried together with the outgoing properties, by making the rdgs:label optional, see below.